spec/models/card_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe Card, type: :model do
+  context 'when validations' do
+    let(:card) { create(:card) }
+
+    it 'when params presence' do
+      expect(card.valid?).to eq(true)
```

Comment:
if you check validation, better to use `build(:card)` instead of `create(:card)` because you actually don't need card in database. `build(:card)` same with `Card.new`